packages/opal-common/opal_common/git/bundle_maker.py

```diff
@@ -53,19 +53,61 @@ def _get_explicit_manifest(self, viewer: CommitViewer) -> Optional[List[str]]:
         loaded into OPA.
 
         This method searches for an explicit manifest file, reads it and returns the list of paths, or None if not found.
-        """
-        def is_manifest_file(f: VersionedFile) -> bool:
-            return f.path == Path(self._manifest_filename)
 
-        manifest_files = list(viewer.files(is_manifest_file))
-        if not manifest_files:
-            return None
+        The manifest file can include references to other directories containing a ".manifest" file,
+        those would be recursively expanded to compile the final manifest list.
+        """
+        visited_paths = []
+
+        def _compile_manifest_file(dir: VersionedDirectory, manifest_file_name: str = ".manifest") -> List[str]:
+            explicit_manifest = list()
+            try:
+                manifest_file = viewer.get_file(dir.path / manifest_file_name)
+                assert manifest_file is not None, "Manifest file not found"
```

Comment:
Rather than raising an `AssertionError` that will be caught on the `except` line below, I feel like it's clearer and simpler to just do an `if` check.